CONFIG.md

```diff
@@ -291,6 +291,30 @@ the properties section in the JSON configuration file. Lastly, the client calls
 
 For information about the path resolution, please refer to the [Path Resolution](#path-resolution-and-object-loading) section.
 
+### Enabling Hazelcast Cloud Discovery
+The purpose of Hazelcast Cloud Discovery is to provide clients to use ip addresses provided by `hazelcast orchestrator`. To enable Hazelcast Cloud Discovery, you should add a discovery token to the `discoveryToken` field and make true the `enabled` field.
```

Comment:
The purpose of Hazelcast Cloud Discovery is to provide clients to use IP addresses provided by `hazelcast orchestrator`. To enable Hazelcast Cloud Discovery, specify a token for the `discoveryToken` field and set the `enabled` field to "true".